### ISSUE TITLE
[magnetdl] Found no results while trying to browse this tracker (Test)

### ISSUE THREAD

**Issue by skuizy** [Jackett/Jackett#5470]
```
Jun 11 21:17:58 odroid jackett[7362]: 06-11 21:17:58 Info Found 0 releases from MagnetDL
Jun 11 21:17:58 odroid jackett[7362]: 06-11 21:17:58 Error System.Exception: Found no results while trying to browse this tracker
Jun 11 21:17:58 odroid jackett[7362]:    at Jackett.Common.Services.IndexerManagerService.TestIndexer(String name) in /home/appveyor/projects/jackett/src/Jackett.Common/Services/IndexerManagerService.cs:line 225
Jun 11 21:17:58 odroid jackett[7362]:    at Jackett.Server.Controllers.IndexerApiController.Test() in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/IndexerApiController.cs:line 128
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Builder.Extensions.UsePathBaseMiddleware.Invoke(HttpContext context)
Jun 11 21:17:58 odroid jackett[7362]:    at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/appveyor/projects/jackett/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 29 System.Exception: Found no results while trying to browse this tracker
Jun 11 21:17:58 odroid jackett[7362]:    at Jackett.Common.Services.IndexerManagerService.TestIndexer(String name) in /home/appveyor/projects/jackett/src/Jackett.Common/Services/IndexerManagerService.cs:line 225
Jun 11 21:17:58 odroid jackett[7362]:    at Jackett.Server.Controllers.IndexerApiController.Test() in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/IndexerApiController.cs:line 128
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
Jun 11 21:17:58 odroid jackett[7362]:    at Microsoft.AspNetCore.Builder.Extensions.UsePathBaseMiddleware.Invoke(HttpContext context)
Jun 11 21:17:58 odroid jackett[7362]:    at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/appveyor/projects/jackett/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 29
```

**Comment by skuizy** [comment]
No details were provided on #5467.

**Comment by garfield69** [comment]
fixed with jackett v0.11.409
